src/app/main/home/screen.test.tsx

```diff
@@ -66,10 +62,13 @@ describe('Home screen', () => {
   });
 
   it('should load the first page of users on the list refresh ', async () => {
-    fireEvent.scroll(usersList, scrollUpEventData);
+    const { refreshControl } = usersList.props;
+    act(() => {
+      refreshControl.props.onRefresh();
```

Comment:
@ekazankova the initial version wasn't correct since we can't to actually test pull-down-to-refresh, so I had to call the list prop explicitly. 

Please check: https://github.com/callstack/react-native-testing-library/issues/809